modules/core/src/v2/coins/dot.ts

```diff
@@ -221,6 +210,27 @@ export class Dot extends BaseCoin {
     return { txHex: signedTxHex };
   }
 
+  /**
+   * Explain/parse transaction
+   * @param params
+   * @param callback
+  */
+  async explainTransaction(params: ExplainTransactionOptions): Promise<accountLib.Dot.TransactionExplanation> {
+    if (!params.txPrebuild.txHex || !params.txPrebuild.validity || !params.txPrebuild.key || !params.txPrebuild.referenceBlock || !params.txPrebuild.version) {
+      throw new Error('missing explain tx parameters');
+    }
+    const txHex = params.txPrebuild.txHex;
+    const sender = new accountLib.Dot.KeyPair({ pub: params.txPrebuild.key }).getAddress();
+    const factory = accountLib.register(this.getChain(), accountLib.Dot.TransactionBuilderFactory);
```

Comment:
I believe DOT also has tokens right? might need a call for `getBaseChain` when `dotTokens` is added in